potofolio.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"> 
+    <title>Document</title>
+</head>
+<body>
+    <div id="header">
+        <div id="logoplaceholder"></div>
+        <button class="menu">
+            <span class="line"></span>
+            <span class="line"></span>
+            <span class="line"></span>
+         </button>
+      
+    </div>
+    <div id="container" >
+        
+        <div id="aboutme" class="intro">
+            <h1 >Hey there.
+            I’m Alex.</h1>
+           <h1 class="job"> I’m a software developer</h1>
```

Comment:
- [ ] The subtitle in your codebase should be implemented using a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`.

- This way will be semantically correct and the screen readers and other technologies parse it. Only one `h1` tag should be in your project 👍